en/docs/integrations/integration-concepts.md

```diff
@@ -0,0 +1,98 @@
+# Concepts
+
+Before you start building integrations with Choreo, there are basic concepts that you need to understand. This section walks you through all the integration concepts you need to know. 
+ 
+## Integration
+
+Integration in Choreo is the process of connecting APIs, events, and streams to function as a single unit that can share information.
+
+
+## Integration Template
+
+An integration template in Choreo is a prebuilt sample designed to address a common integration requirement. Choreo provides a comprehensive set of integration templates for you to select and use depending on your requirement.
+
+
+## Trigger
+
+A trigger in Choreo is an event or an action that can cause a Choreo application to start executing.
+
+Choreo supports the following trigger types:
+
+ - **API**: Allows you to expose an application as an API endpoint. 
```

Comment:
In the new UI we don't have this trigger now

Reply:
Removed from doc.